app/access/api.py

```diff
@@ -42,3 +45,30 @@ def users(request: HttpRequest) -> dict[str, list[UserSchema]]:
     models = User.objects.all()
     responses = [user_to_response(model) for model in models]
     return {"items": responses}
+
+
+@router.post("users", response={201: UserSchema})
+def create_user(request: HttpRequest, user_in: UserSchema) -> UserSchema:
+    """Create the given user and return it.
+
+    Return HTTP status code
+
+        - 201 (Created) if the User was created as expected
+        - 404 (Not Found) if there is no provider with the given provider ID
+        - 409 (Conflict) if there is already a record with the same username
+        - 422 (Unprocessable Content) if there is any other invalid value
```

Comment:
I am not really sure either. I wouldn't say it makes clients' life substantially complicated but it might be work for nothing on our side to distinguish these cases.

I keep it as it is for the moment and add it as a point to discuss.